config/gulp/translation.js

```diff
@@ -0,0 +1,71 @@
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var del = require('del');
+var pkg = require('../../package.json');
+var runSequence = require('run-sequence');
+var spawn = require('cross-spawn');
+var rename = require('gulp-rename');
+var replace = require('gulp-replace');
+var stateNames = require('./state-names.json');
+
+gulp.task('clean-translation', function () {
+  gutil.log(gutil.colors.cyan('clean-translation'), 'Removing generated register/ files');
+  return del([
+    './layouts/registrar',
+    './content/registrar',
+  ]);
+});
+
+gulp.task('copy-content-spanish', function (done) {
+
+  return gulp.src('./content/register/*.md')
+    .pipe(rename(function (path) {
+      var file = stateNames[path.basename].file;
+      path.basename = file;
+    }))
+    .pipe(replace(/title = "(.+)"/, function (match, p1) {
+      var name = p1.replace(/\s/g, '-').replace(/\./g, '').toLowerCase();
+      var title = stateNames[name].title;
+      var file = stateNames[name].file;
+      return (
+        'title = "' + title + '"' +
+        "\n" +
+        'slug = "' + file + '"'
```

Comment:
Replacing the `title` and adding a new line and `slug = "*"` line in the newly generated content.